subsys/bluetooth/common/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ zephyr_library()
 
 zephyr_library_sources(addr.c)
 zephyr_library_sources(dummy.c)
-zephyr_library_sources(log.c)
```

Comment:
The addition must be in the previous commit. The deletion should be in this commit.